### PR TITLE
Landing

### LINKED ISSUE #3: Frontend: Landing Page
Figma gives an example using ACM Teach LA. This needs to be generalized to allow for other committees to use this template too. Navigation bar is not part of this issue

### PR DESCRIPTION
**Pull request by caz07**
Created a landing page

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,76 +1,33 @@
 import type { NextPage } from 'next';
 import MainLayout from '../components/MainLayout';
-import styles from '../styles/Home.module.scss';
+import styles from '../styles/landing.module.scss';
```
Comment (ChiweiChen) [inline pages/index.tsx:3]: this is a comment!

File: styles/landing.module.scss [hunk 2]
```diff
@@ -0,0 +1,127 @@
+$committee-color: #5eda12;
+
+.masthead {
+  background-image: url("../public/Illustration.png");
```
Comment (nendow02) [inline styles/landing.module.scss:4]: Can we delete the crop image from our project since we are using the png instead

**Review by nendow02 — DISMISSED** [review]
One small nit but that’s it

**Review by yinuj — APPROVED** [review]
Looks good to me! Thanks for the change.

**Review by ChiweiChen — APPROVED** [review]
Everything looks good to me!

**Pull request merged**